src/index.js

```diff
@@ -47,6 +48,16 @@ class Root extends React.Component<Props, State> {
     return Promise.resolve(user);
   };
 
+  onSocialLogin = async provider => {
+    const authUrl = await socialLogin(provider);
+
+    if (authUrl) {
```

Comment:
it shouldn't be `if(!authUrl)` ?

Reply:
it's still not working locally, but... 🤦‍♂️  🤦‍♂️